nemo/collections/nlp/data/text_classification/text_classification_descriptor.py

```diff
@@ -12,43 +12,39 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 from typing import List
 
 from nemo import logging
 from nemo.collections.nlp.data.data_utils.data_preprocessing import (
     fill_class_weights,
     get_freq_weights,
     get_label_stats,
-    if_exist,
 )
 
 __all__ = ['TextClassificationDataDesc']
 
 
 class TextClassificationDataDesc:
-    def __init__(self, data_dir: str, modes: List[str] = ['train', 'test', 'dev']):
+    def __init__(self, train_file: str, val_files: List[str] = []):
```

Comment:
your validation in the text_classification model currently only supports a single file, maybe turn val_files into val_file and simply the code here?

Reply:
But I should support both test and validation files.